src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -306,8 +223,18 @@ function setupWithdrawalAccount(params) {
                 setFreePlanVerifiedBankAccountID(responseACHData.bankAccountID);
             }
 
-            if (currentStep === CONST.BANK_ACCOUNT.STEP.REQUESTOR) {
-                checkDataAndMaybeStayOnRequestorStep(responseACHData, getNextStep(updatedACHData));
+            const nextStep = getNextStep(updatedACHData);
+
+            // Some steps require several substeps based on some data.
+            // This logic is done php side: when we need to display a specific step with specific data, php add them in the response in nextStepValues
+            // Example 1: When forcing manual step after adding Chase bank account via Plaid, so we can ask for the real numbers instead of the plaid substitutes
+            // Example 2: When on the requestor step, showing Onfido view after submitting the identity and retrieving the sdkToken
```

Comment:
Has the information about whether or not a user needs to do onFido always been determined/processed/returned by php?

Reply:
no, but now it will be since https://github.com/Expensify/Web-Secure/pull/2121